receiver/README.md

```diff
@@ -164,6 +164,18 @@ receivers:
       fetch_endpoint: "jaeger-collector:1234"
 ``` 
 
+Remote sampling can also be directly served by the collector by providing a sampling json file.
+
+```yaml
+receivers:
+  jaeger:
+    protocols:
+      grpc:
+    remote_sampling:
```

Comment:
Since grpc is required wouldn't it make more sense to have this as one option under grpc? Or does it make the code much more complex?

Reply:
I considered it, but chose against it because currently the Config struct treats all protocols the same:
```
Protocols      map[string]*receiver.SecureReceiverSettings `mapstructure:"protocols"`
```
Also there is the custom unmarshalling logic that would be complicated by this.  I could play around with some options, but I'm unsure how that clean I could get it.

Reply:
I see that's what I was afraid of :)